com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfJoints/UrdfJointRevolute.cs

```diff
@@ -101,16 +101,7 @@ protected override void OnUpdateJointState(float deltaState)
         protected override void ImportJointData(Joint joint)
         {
             AdjustMovement(joint);
-            if (joint.dynamics != null)
-            {
-                unityJoint.angularDamping = (double.IsNaN(joint.dynamics.damping)) ? defaultDamping : (float)joint.dynamics.damping;
-                unityJoint.jointFriction = (double.IsNaN(joint.dynamics.friction)) ? defaultFriction : (float)joint.dynamics.friction;
-            }
-            else
-            {
-                unityJoint.angularDamping = 0;
-                unityJoint.jointFriction = 0;
-            }
+            SetDynamics(joint.dynamics);
```

Comment:
nice refactor

Reply:
thanks! may need another approval due to the stale commits